BackendApartmentReservation/BackendApartmentReservation/Controllers/EmployeesController.cs

```diff
@@ -24,11 +25,13 @@ public EmployeesController(IEmployeeManager employeeManager)
         [Route("employees")]
         public async Task<IActionResult> Register([FromBody] RegisterRequest model)
         {
-            var employeeId = await _employeeManager.CreateEmployee(
-                             model.FirstName,
-                             model.LastName,
-                             model.Email,
-                             model.Office);
+            DbEmployee dbEmployee = new DbEmployee();
+            dbEmployee.FirstName = model.FirstName;
+            dbEmployee.LastName = model.LastName;
+            dbEmployee.Email = model.Email;
+            dbEmployee.Office.Address = model.Office;
```

Comment:
This will 100% end up in `NullReferenceException`. You must create office first, and only then assign address